source/gui/__init__.py

```diff
@@ -480,11 +480,32 @@ def onInstallCommand(self, evt):
 
 		installerGui.showInstallGui()
 
+	CRFT_INTRO_MSG: str = _(
```

Comment:
Please expand "MSG" to "MESSAGE". I think we might as well make this private since I see no reason it needs to be public.
```suggestion
	CRFT_INTRO_MESSAGE: str = _(
```